### PR TITLE
Add memEx

### PR DESCRIPTION
**Pull request by shibaobun**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/memex.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "memEx"
+website_url: "https://gitea.bubbletea.dev/shibao/memEx"
+source_code_url: "https://gitea.bubbletea.dev/shibao/memEx"
+description: "A structured, zettlekasten and org-mode inspired memex."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+  - Elixir
```
Comment (Ki-er) [inline software/memex.yml:9]: > https://gitea.bubbletea.dev/shibao/memEx#installation

```suggestion
platforms:
  - Docker
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick change

File: software/memex.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: "memEx"
+website_url: "https://gitea.bubbletea.dev/shibao/memEx"
+source_code_url: "https://gitea.bubbletea.dev/shibao/memEx"
+description: "A structured, zettlekasten and org-mode inspired memex."
```
Comment (Ki-er) [inline software/memex.yml:4]: This could be slightly tweaked to explain what a `memex` is in plain English so that people looking for a project but might not know what it is, can be interested.

**Review by Ki-er — CHANGES_REQUESTED** [review]

**Review by nodiscc — CHANGES_REQUESTED** [review]
@shibaobun the description should explain what the software does, see the other review above.

**Review by nodiscc — CHANGES_REQUESTED** [review]
minor adjustment

File: software/memex.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: "memEx"
 website_url: "https://gitea.bubbletea.dev/shibao/memEx"
 source_code_url: "https://gitea.bubbletea.dev/shibao/memEx"
-description: "A structured, zettlekasten and org-mode inspired memex."
+description: "A structured, zettlekasten and org-mode inspired personal knowledge base."
```
[the remark this answers is not in the document]
Comment (shibaobun) [reply software/memex.yml:4]: Thank you for the help, this is much more clear! :smile:

**Review by nodiscc — APPROVED** [review]
